bundler/man/bundle-install.ronn

```diff
@@ -151,11 +169,16 @@ time `bundle install` is run, use `bundle config` (see bundle-config(1)).
   in the remembered list of groups given to --without, it is removed
   from that list.
 
+  This option is deprecated in favor of the `with` setting.
+
 * `--without=<list>`:
   A space-separated list of groups referencing gems to skip during installation.
   If a group is given that is in the remembered list of groups given
   to --with, it is removed from that list.
 
+  This option is deprecated in favor of the `without` setting.
+
+
```

Comment:
Is the extra line space needed? otherwise I would remain consistent and only have the 1 line space.

Reply:
Ooops, my bad. Let me update this!